tests/test_evaluation_suite.py

```diff
@@ -0,0 +1,18 @@
+from unittest import TestCase
+
+from evaluate import EvaluationSuite
+
+
+class TestEvaluationSuite(TestCase):
+    def setUp(self):
+        self.evaluation_suite = EvaluationSuite.load("evaluate/evaluation-suite-ci")
+
+    def test_running_evaluation_suite(self):
+        results = self.evaluation_suite.run("lvwerra/distilbert-imdb")
```

Comment:
Would you mind replacing this with a dummy model like in the evaluators? The tests are already very slow and we test that loading from the hub works already in the evaluator tests.

Reply:
Done!